config.go

```diff
@@ -5,11 +5,13 @@
 
 package main
 
+/*
```

Comment:
Is this file kept for reference until the PR is finished?

Reply:
Yes